src/redux/actions/actionCreators/articles.js

```diff
@@ -0,0 +1,46 @@
+import '@babel/polyfill';
+import axios from 'axios';
+import {
+  CREATE_ARTICLE, GET_ARTICLE, FAILED_ARTICLE_CREATION,
+  FAILED_ARTICLE_UPDATE, UPDATE_ARTICLE,
+} from '../actionTypes';
+
+export const publishArticle = (articleData) => async (dispatch) => {
+  try {
+    const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjIwYjlhZGQ4LTM1ZTUtNDgwNy04M2FlLWZlY2Q1YjAxMjNhOSIsImVtYWlsIjoiY2VsZXN0aW40M0BnbWFpbC5jb20iLCJ1c2VybmFtZSI6Im1hcmxleSIsImlhdCI6MTU2NjU3MTcwNywiZXhwIjoxNTY2NjU4MTA3fQ.EK6hSflXBLL6TqoxmPdRhmrG4-tlPl2Kaa7McQ0BAAc';
+    const article = await axios.post('http://127.0.0.1:3000/api/articles', articleData, { headers: { Authorization: `Bearer ${token}` } });
```

Comment:
kindly donot hardcode `urls` in your project it would be better if you make use of the `.env` file